tests/h/search/index_test.py

```diff
@@ -120,8 +268,25 @@ def _get(annotation_id):
         return _get
 
     @pytest.fixture
-    def search(self, es_client):
-        return elasticsearch1_dsl.Search(using=es_client.conn).fields([])
+    def AnnotationTransformEvent(self, patch):
+        return patch('h.search.index.AnnotationTransformEvent')
+
+
+class TestDelete(object):
+    def test_annotation_is_marked_deleted(self, es_client, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id")
+
+        index(annotation)
+        result = es_client.conn.get(index=es_client.index,
```

Comment:
We're going to need to abstract the operation of fetching an annotation from the index in future in order to be able to run tests against ES 1 and ES 6.2.